src/include/nodes/parsenodes.h

```diff
@@ -3100,6 +3100,14 @@ typedef struct RenameStmt
 	bool		missing_ok;		/* skip error if missing? */
 } RenameStmt;
 
+typedef struct AlterModuleRenameStmt
```

Comment:
The other structures in this section are preceded by a comment; should probably do the same for AlterModuleRenameStmt